portfolio/src/main/webapp/blog.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: auto;
+  border: 1px solid green;
+  padding: 4vh 2.5vw 4vh 2.5vw;
+  font-family: 'Montserrat', serif;
+}
+
+#article {
+  border: 1px solid gold;
+  padding: 1.5vh .9375vw 1.5vh .9375vw;
+}
+
+.header {
+  border: 1px solid black;
+}
+
+.title {
+  margin-top: 1vh;
+  border: 1px solid purple;
+  width: 100%;
```

Comment:
Instead of setting width to 100%, you could also set the `display` field, which is a little more semantic and easier for others to read. `block` and `inline-block` both work, but in context of your class name here, I'm guessing you'd want `inline-block`